tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = flake8, pylint, check-{py27,py36}
+envlist = flake8, pylint, check-{py27,py36,py37}
```

Comment:
Please move the python 3.7 test to a different PR. It would also be nice to test against Python 3.8.

Reply:
Done.